features/system-report/src/main/java/org/opennms/systemreport/system/OSReportPlugin.java

```diff
@@ -129,6 +150,55 @@ public Map<String, Resource> getEntries() {
             map.put("Description", map.remove("Distribution Description"));
         }
 
+        String hostName = m_ipInterfaceDao.findAll().stream().map(OnmsIpInterface::getIpHostName).distinct().collect(Collectors.joining(","));
+        String ipAddress = m_ipInterfaceDao.findAll().stream().map(s->s.getIpAddress().getHostAddress()).distinct().collect(Collectors.joining(","));
+
+        map.put("Host Name",getResource(hostName));
+        map.put("Ip Address",getResource(ipAddress));
```

Comment:
we don't need hostName, ipAddress from all Ip Interfaces. 

The ask was to get IPAddress and HostName for OpenNMS system.
For now, you can remove these two things.